src/runtime/nuxt-image-mixins.ts

```diff
@@ -65,6 +65,7 @@ export default {
   },
   data () {
     return {
+      error: '',
```

Comment:
Is the type object or string? :D

Reply:
I store error message not whole error object `this.error = e.message`.
Do we need error object? 🤔